packs/data/feats.db/impaling-thrust.json

```diff
@@ -10,7 +10,7 @@
             "value": 2
         },
         "description": {
-            "value": "<p><strong>Requirements</strong> You are wielding a melee weapon that deals piercing damage.</p>\n<hr />\n<p>You impale your enemy and hold them in place with your weapon.</p>\n<p>Make a melee Strike with the required weapon. If the Strike hits and deals damage, your target is @UUID[Compendium.pf2e.conditionitems.Grabbed]{Grabbed} until they successfully @UUID[Compendium.pf2e.actionspf2e.Escape]{Escape}, you attack with the required weapon, or you @UUID[Compendium.pf2e.actionspf2e.Release]{Release} the required weapon, whichever comes first.</p>\n<p>When the target is no longer grabbed, they take @UUID[Compendium.pf2e.conditionitems.Persistent Damage]{Persistent Bleed Damage} equal to the number of weapon damage dice.</p>"
+            "value": "<p><strong>Requirements</strong> You are wielding a melee weapon that deals piercing damage.</p>\n<hr />\n<p>You impale your enemy and hold them in place with your weapon. Make a melee Strike with the required weapon. If the Strike hits and deals damage, your target is @UUID[Compendium.pf2e.conditionitems.Grabbed]{Grabbed} until they successfully @UUID[Compendium.pf2e.actionspf2e.Escape]{Escape}, you attack with the required weapon, or you Release the required weapon, whichever comes first. When the target is no longer grabbed, they take persistent bleed damage equal to the number of weapon damage dice.</p>"
```

Comment:
Should this include a leveled bleed damage link?

Reply:
Not really doable since it's based on the weapon damage of the strike.